server/app/services/applications/ProgramAdminApplicationService.java

```diff
@@ -227,6 +227,9 @@ public void setNote(ApplicationModel application, NoteEvent note, AccountModel a
     ApplicationEventModel event =
         new ApplicationEventModel(application, Optional.of(admin), details);
     eventRepository.insertSync(event);
+    // save the latest note on the applications table too
+    application.setLatestNote(note.note());
+    application.save();
   }
```

Comment:
Two thoughts:
- This needs to be done in the same transaction as the application event or they could get out of sync if there's a DB error partway through, or a race condition between two users.
- I'd actually move the whole `setNote` method into the `ApplicationEventsRepository` and have that be the only way to set notes on an application.

Reply:
done!